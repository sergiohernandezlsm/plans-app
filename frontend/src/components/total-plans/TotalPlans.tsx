import Row from 'react-bootstrap/Row';
import Col from 'react-bootstrap/Col';
import Card from 'react-bootstrap/Card';
import Container from 'react-bootstrap/Container';
import styles from './TotalPlans.module.css';

interface Props {
  totalPlans: number;
}

const TotalPlans: React.FC<Props> = ({ totalPlans }) => {
  return (
    <Card className={styles['card-styles']}>
      <Card.Body>
        <Container>
          <Row>
            <Col className={`${styles['center-item']} ${styles['extra-style-item']}`} xs={12} sm={10}>
              <Card.Text className={styles['total-text-styles']} >TOTAL SUBSCRIPTION PLANS:</Card.Text>
            </Col>
            <Col className={styles['center-item']} xs={12} sm={2}>
              <Card.Text className={styles['total-cost-styles']}>Â£{totalPlans}</Card.Text>
            </Col>
          </Row>
        </Container>
      </Card.Body>
    </Card >
  )
}

export default TotalPlans;