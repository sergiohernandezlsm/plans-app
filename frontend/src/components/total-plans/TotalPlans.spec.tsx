import { shallow } from 'enzyme';
import TotalPlans from './TotalPlans';

const props = {
  totalPlans: 1
}

describe('<MyComponent />', () => {
  it('renders <TotalPlans /> component', () => {
    const wrapper = shallow(<TotalPlans {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});