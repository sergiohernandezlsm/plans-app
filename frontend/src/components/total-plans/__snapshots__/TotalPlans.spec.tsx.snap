// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyComponent /> renders <TotalPlans /> component 1`] = `ShallowWrapper {}`;
